### PR TITLE
Rencent blocks section on the landing page completed

### PR DESCRIPTION
**Pull request by JM066**

### CODE REVIEW

**Comment by JM066** [comment]
I didn't make a component for the recent blocks table cos it only gets repeated once. Do you still recommend to make a component?

File: src/components/Table/HashTimeCell/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  time,
+  identicon,
+  index,
+  link,
+  hashLeft,
+  hashRight,
+  className,
+}: Props) {
+  const shortHash = `${hash.substring(0, hashLeft)}...${hash.substring(
+    hash.length - hashRight!
```
Comment (BruceJi7) [inline src/components/Table/HashTimeCell/index.tsx:30]: Maybe there is a better solution than ! in this case. Maybe it should default to 0 instead?

File: src/components/Table/HashTimeCell/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import styles from "./HashTimeCell.module.scss";
+
+interface Props {
+  hash: string;
+  time: string;
+  identicon?: boolean;
+  index: number;
+  link: string;
+  hashLeft?: number | undefined;
+  hashRight?: number | undefined;
```
Comment (BruceJi7) [inline src/components/Table/HashTimeCell/index.tsx:16]: The ? already covers undefined

File: src/store/store.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+//   setSocketData: (newData) =>
+//     set({
+//       socketData: {
+//         activityBlocks: newData.newBlocks,
+//         newTxns: newData.newTxns,
+//       },
+//     }),
+// }));
+
+// export default useStore;
```
Comment (BruceJi7) [inline src/store/store.ts:19]: What is going on here lol

**Review by BruceJi7 — COMMENTED** [review]
One component/function could be changed, otherwise good.
Also the store hmm delete it? Or uncomment it but leave it empty maybe

File: src/components/Table/HashTimeCell/index.tsx (hunk 1 above)
Comment (JM066) [reply src/components/Table/HashTimeCell/index.tsx:30]: Thanks. Yeah, that's much better! Fixed!

File: src/components/Table/HashTimeCell/index.tsx (hunk 2 above)
Comment (JM066) [reply src/components/Table/HashTimeCell/index.tsx:16]: Corrected! Thanks

File: pages/txns/index.tsx [hunk 4]
```diff
@@ -1,2 +1,14 @@
 import Txns from "../../src/pages/Txns";
+import { getLatestTxns } from "../api/index";
 export default Txns;
+
+export async function getStaticProps() {
```
Comment (JM066) [inline pages/txns/index.tsx:5]: Try ServerSide here

File: src/components/Layout/index.tsx [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+import Loading from "../Loading";
+import Disconnected from "../Disconnected";
 import PageContainer from "./PageContainer";
 
-import useChannelHash from "@/hooks/useChannelHash";
-
-import Loading from "../Loading";
-// import Disconnected from "../Disconnected";
+import useLatestBlocksData from "@/hooks/useChannelHash";
+// import useIsMobile from "@/hooks/useIsMobile";
```
Comment (JM066) [inline src/components/Layout/index.tsx:10]: remove comment

File: src/components/Layout/index.tsx [hunk 6]
```diff
@@ -16,10 +16,11 @@ interface Layout {
 }
 
 function Layout({ children }: Layout) {
-  const { channelHash, loadingChannelHash } = useChannelHash();
+  const { channelHash, loadingChannelHash } = useLatestBlocksData();
+  // const isMobile = useIsMobile();
```
Comment (JM066) [inline src/components/Layout/index.tsx:20]: remove comment

File: src/components/Table/TxnsCell/index.tsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+import styles from "./TxnsCell.module.scss";
+
+function TxnsCell({
+  txcount,
+  className,
+}: {
+  txcount: number;
+  className?: string;
+}) {
+  const singlularOrPlural = txcount > 1 ? "Transactions" : "Transaction";
```
Comment (JM066) [inline src/components/Table/TxnsCell/index.tsx:16]: singular

File: src/helpers/index.tsx [hunk 8]
```diff
@@ -15,21 +22,109 @@ export function getContractType(serviceType: string | undefined) {
       return "basic";
   }
 }
+export const getTxnsIcon = (action: string) => {
+  switch (action) {
+    case "conx":
```
Comment (JM066) [inline src/helpers/index.tsx:27]: default covers this case :O

**Pull request merged**